build.gradle

```diff
@@ -183,17 +195,13 @@ dependencyCheck {
   }
 }
 
-repositories {
-  mavenLocal()
-  mavenCentral()
-  maven { url 'https://jitpack.io' }
-}
-
 ext {
   log4JVersion = '2.19.0'
```

Comment:
Should this be changed to 2.20.0 to match the version of the log4j-slf4j2-impl dependency?  If so, also change the log4j-slf4j2-impl dependencies to use this property instead of a hard coded value.

Reply:
done